src/emitterstate.cpp

```diff
@@ -160,6 +160,12 @@ void EmitterState::EndedGroup(GroupType::value type) {
     return SetError(ErrorMsg::UNEXPECTED_END_MAP);
   }
 
+  // if have the unhandled Tag/Anchor, set the error message
+  if (m_hasTag)
```

Comment:
Put {} around the body of this if statement, even though it's only one line.

(I've been meaning to go through and enforce that throughout the codebase, but haven't had the chance.)

Reply:
done. Maybe I can change the clang-format Configuration File && Optimization Option, so we can clang-format all files quickly.